simulation_parameters/common/input_options.json

```diff
@@ -0,0 +1,148 @@
+[
+    {
+        "EnvironmentId": [
+            "Game"
+        ],
+        "GroupName": "Movement",
+        "Actions": [
+            {
+                "InputName": "g_move_left",
+                "DisplayName": "Move left"
```

Comment:
I think it would make translations easier if these were named "Name" to be consistent how the name is used in the other json files.

Reply:
Should I rename the InputName to name or the DisplayName to name?

Reply:
I'd say it makes more sense to make the translation be based on DisplayName

Reply:
![grafik](https://user-images.githubusercontent.com/30809803/97805219-141a5500-1c55-11eb-8077-85348fd260bf.png)
should I still rename?

Reply:
If you make that separate class that is loaded from the JSON there's not going to be a problem with that.